src/components/interaction-designer/resource-editors/PerLanguageResourceEditor.vue

```diff
@@ -22,17 +21,15 @@
 <script lang="ts">
 import Vue from 'vue'
 import {namespace} from 'vuex-class'
-import {
-  IBlock,
-  IFlow,
-} from '@floip/flow-runner'
+import {IBlock, IFlow, ILanguage} from '@floip/flow-runner'
 import Lang from '@/lib/filters/lang'
 import Permissions from '@/lib/mixins/Permissions'
 import Routes from '@/lib/mixins/Routes'
 import FlowUploader from '@/lib/mixins/FlowUploader'
 import {Component, Prop} from 'vue-property-decorator'
 import {mixins} from 'vue-class-component'
 import {TabsPlugin} from 'bootstrap-vue'
+import {orderLanguages} from '@/store/flow/flow'
```

Comment:
again, the `flow.languages` are unsorted (and we should not rely on its order). But we are sorting them here - in the UI - to display in certain order

Reply:
Can we keep this comment in the code, I feel like that would be useful in the future?

Reply:
Sure, I will add it 👍

Reply:
Added here now: https://github.com/FLOIP/flow-builder/pull/385/commits/3b2162286327bb03f8cecdfd9ca0e1266e62cc2c
(I didn't rebuild the project because it's only a comment, it will get picked up at the next rebuild)